### PR TITLE
added Chanakya's Chakkravyuh

### PR DESCRIPTION
**Pull request by 404Nikhil**

### CODE REVIEW

**Comment by 404Nikhil** [comment]
Please check this PR @edm00se

File: readme.md [hunk 1]
```diff
@@ -2256,6 +2256,18 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 4 |       5  |  20m   |
 
+### [Chanakya's Chakkravyuh](https://boardgamegeek.com/boardgame/154680/chanakyas-chakkravyuh)
+
+> Chanakya's Chakkravyuh is a unique learning tool in the form of a financial board game. Its main purpose is to help players from all walks of life (including common man, entrepreneurs, corporate professionals, students, educators, home makers, investors) attain financial literacy. Minimum age is 15 years and above.
+Inspired by the 2000-year old wisdom of prominent Indian economist Chanakya, this game enlightens individuals on how to manage money and connect financial knowledge to resourceful opportunities, which are ever-present in today's changing world.
+The player initially chooses a dream out of various dreams listed on the board. Each dream has a purchase price listed on it. In order to win the game, the player must purchase his/her pre-selected dream.
```
Comment (edm00se) [inline readme.md:2263]: These lines, as part of the description block, should also be quoted. This is what the linter rule is currently complaining about.

```suggestion
> Inspired by the 2000-year old wisdom of prominent Indian economist Chanakya, this game enlightens individuals on how to manage money and connect financial knowledge to resourceful opportunities, which are ever-present in today's changing world.
> The player initially chooses a dream out of various dreams listed on the board. Each dream has a purchase price listed on it. In order to win the game, the player must purchase his/her pre-selected dream.
```

File: readme.md [hunk 2]
```diff
@@ -2256,6 +2256,18 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+![Chanakya's Chakkravyuh](https://www.chanakyaschakkravyuh.com/wp-content/uploads/2016/01/LC-Box-Front-1-300x220.jpg)
```
Comment (edm00se) [inline readme.md:2265]: Please update the image source to use one from preferably wikipedia or BGG. While the publisher may be an authoritative source, there's no guarantee their wordpress site uses an image cdn, unlike the other two mentioned.

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update the description b lock and image source.

**Comment by 404Nikhil** [comment]
Hello @edm00se so I have fixed the cause of linting error and about the image source in BGM it's was not allowing me to resize the image size in URL itself and there's no existing Wikipedia page for it. So I decided to simply download one put the images in assets folder and link the relative path. It's previewing fine. Hopefully no error come up this time.

**Comment by edm00se** [comment]
I'd love to not need to host the image,  but a quick search led me to the same conclusion. I'll have another look this evening when I have a few minutes and see if I can find another that will do what I'd like first.

**Pull request closed without merging**